colin/core/result.py

```diff
@@ -97,6 +99,44 @@ def save_json_to_file(self, file):
                   fp=file,
                   indent=4)
 
+    @property
+    def xunit(self):
+        """
+        Get the xunit representation of results
+
+        :return: str
+        """
+
+        top = Element("testsuites")
+
+        testsuite = SubElement(top, "testsuite")
+        top.append(testsuite)
+
+        for r in self.results:
+            testcase = SubElement(testsuite, "testcase", {
+                'name': r.check_name,
+                'status': "PASSED" if r.ok else "FAILED", # Can't use PASSED or FAILED global variables because their values are PASS and FAIL respectively.
+                'url': r.reference_url
+            })
+            testsuite.append(testcase)
+            logs = SubElement(testcase, "logs")
+            for log in r.logs:
+                log = SubElement(logs, "log", {
+                    'message': log,
+                    'result': 'INFO',
+                    'waiver_authorization': 'Not Waivable'
+                })
+                logs.append(log)
+
+        rough_string = ElementTree.toString(top, 'utf-8')
+        reparsed = minidom.parseString(rough_string)
+        return reparsed.toprettyxml(indent="  ")
+
+    def save_xunit_to_file(self, file):
+        f = open(file, 'w')
+        f.write(self.xunit)
+        f.close()
```

Comment:
You don't need to use `open`. The object you get from `click` is already writable.

Reply:
Thanks